### PR TITLE
RFS-225 - renamed firstFork as orchid

### PR DESCRIPTION
**Pull request by josedahlquist**
:robot: **build using** *utils:RFS-225* *fed:RFS-225*

### CODE REVIEW

**Comment by rskops** [comment]
SonarQube analysis reported 2 issues

1. ![INFO][INFO] [MainNetConfig.java#L36](https://github.com/rsksmart/rskj/blob/f03568edb4d3d4fb9c8a65b018c7f2f5b943a332/rskj-core/src/main/java/org/ethereum/config/net/MainNetConfig.java#L36): Complete the task associated to this TODO comment. [rule](https://jenkins.rsklabs.io/sonar/coding_rules#rule_key=squid%3AS1135)
1. ![INFO][INFO] [TestNetConfig.java#L35](https://github.com/rsksmart/rskj/blob/f03568edb4d3d4fb9c8a65b018c7f2f5b943a332/rskj-core/src/main/java/org/ethereum/config/net/TestNetConfig.java#L35): Complete the task associated to this TODO comment. [rule](https://jenkins.rsklabs.io/sonar/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: rskj-core/src/main/java/org/ethereum/config/blockchain/HardForkActivationConfig.java [hunk 1]
```diff
@@ -20,18 +20,17 @@
 import com.typesafe.config.Config;
 
 public class HardForkActivationConfig {
-    private final int firstFork;
+    private final int orchid;
```
Comment (amendelzon) [inline rskj-core/src/main/java/org/ethereum/config/blockchain/HardForkActivationConfig.java]: Maybe `orchidActivationHeight`?

Comment (aeidelman) [reply rskj-core/src/main/java/org/ethereum/config/blockchain/HardForkActivationConfig.java]: Agree, not very clear without reading the rest of the code.

**Comment by rskops** [comment]
SonarQube analysis reported 2 issues

1. ![INFO][INFO] [MainNetConfig.java#L36](https://github.com/rsksmart/rskj/blob/c27e1506efdf82e56b77f231bcbc924b9ce1aa0a/rskj-core/src/main/java/org/ethereum/config/net/MainNetConfig.java#L36): Complete the task associated to this TODO comment. [rule](https://jenkins.rsklabs.io/sonar/coding_rules#rule_key=squid%3AS1135)
1. ![INFO][INFO] [TestNetConfig.java#L35](https://github.com/rsksmart/rskj/blob/c27e1506efdf82e56b77f231bcbc924b9ce1aa0a/rskj-core/src/main/java/org/ethereum/config/net/TestNetConfig.java#L35): Complete the task associated to this TODO comment. [rule](https://jenkins.rsklabs.io/sonar/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**